3.11/release-notes-new-features311.md

```diff
@@ -2,15 +2,34 @@
 layout: default
 description: ArangoDB v3.11 Release Notes New Features
 ---
-Features and Improvements in ArangoDB 3.11
-==========================================
+# Features and Improvements in ArangoDB 3.11
 
 The following list shows in detail which features have been added or improved in
 ArangoDB 3.11. ArangoDB 3.11 also contains several bug fixes that are not listed
 here.
 
-AQL
----
+## Analyzers
+
+### `geo_s2` Analyzer (Enterprise Edition)
+
+This new Analyzer lets you index GeoJSON data with inverted indexes or Views
+similar to the existing `geojson` Analyzer, but it internally uses a format for
+storing the geo-spatial data that is more efficient.
+
+You can choose between different formats to make a tradeoff between the size on
+disk, the precision, and query performance:
+
+- 8 bytes per coordinate using 4-byte integer values, with limited precision.
+- 16 bytes per coordinate using 8-byte floating-point values, which is still
+  more compact than the VelocyPack format used by the `geojson` Analyzer
+- 24 bytes per coordinate using the native Google S2 format to reduce the number
+  of computations necessary when you execute geo-spatial queries.
```

Comment:
or lat lng pair, point or something, coordinate sounds like single lat/lon

```suggestion
- 8 bytes per vertex using 4-byte integer values, with limited precision.
- 16 bytes per vertex using 8-byte floating-point values, which is still
  more compact than the VelocyPack format used by the `geojson` Analyzer
- 24 bytes per coordinate using the native Google S2 format to reduce the number
  of computations necessary when you execute geo-spatial queries.
```

Reply:
The GeoJSON spec calls it a _position_ but doesn't state whether latitude and longitude values are coordinates on their own or only in combination. There appear to be different opinions on the web. I don't like the term _vertex_ in this context because it's not used anywhere else to refer to positions in the docs and is also ambiguous (geometry vs. geo-spatial vs. graph). I initially used the term _pair_, so we could potentially change it to _longitude-latitude pair_ or _coordinate pair_...

Reply:
My suggestion is to go for "coordinate pair". That's how it is mainly expressed in the geo-spatial terminology.

Reply:
Do you forgot to update it? I don't see "outdated" mark

Reply:
Good catch. I updated it for 3.10 but not 3.11 (doesn't show in a local diff between folders). Fixed.